samples/net/openthread/shell/prj.conf

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=4096
+
+# Enable Networking and OpenThread stack
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_OPENTHREAD=y
+
+# Disable certain parts of Zephyr IPv6 stack
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
```

Comment:
I wonder if we needs this, OT interfaces should now disable MLD/ND at runtime with `NET_IF_IPV6_NO_ND` and `NET_IF_IPV6_NO_MLD` flags. If we want to optimize image size we can disable it here, but I guess we could also skip this if keeping the config simple is the priority. I'll leave it up to you to decide.

Reply:
Yo are right, it works fine without these options, so I removed them. Originally took them from the `coprocessor` sample.